view/templates/intros.tpl

```diff
@@ -9,7 +9,7 @@
 {{if $gender}}<dl><dt>{{$gender_label}}</dt><dd>{{$gender}}</dd></dl>{{/if}}
 {{if $keywords}}<dl><dt>{{$keywords_label}}</dt><dd>{{$keywords}}</dd></dl>{{/if}}
 {{if $about}}<dl><dt>{{$about_label}}</dt><dd>{{$about}}</dd></dl>{{/if}}
-<div class="intro-knowyou">{{$knowyou}}</div>
+<div class="intro-knowyou">{{$lbl_knowyou}} {{$knowyou}}</div>
```

Comment:
>  Is it more "label" or more "lbl" or a third form

I don't really know it. Friendica isn't really unique in this area.

At the moment I try to reduce code duplication in notifications.php. Maybe another commit will follow before fixing the label thing